<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMsr2DataDialog</class>
 <widget class="QDialog" name="PMsr2DataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>msr2data input</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../musredit.qrc">
    <normaloff>:/images/musrmsr2data.xpm</normaloff>:/images/musrmsr2data.xpm</iconset>
  </property>
  <widget class="QGroupBox" name="fRunListInput_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Run List Input</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>531</width>
      <height>97</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="fFirst_label">
         <property name="text">
          <string>First</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fFirst_lineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fLast_label">
         <property name="text">
          <string>Last</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fLast_lineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fOrFirstLast_label">
         <property name="text">
          <string>&lt;b&gt;OR&lt;/b&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="fRunList_label">
         <property name="text">
          <string>Run List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fRunList_lineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fOrRunList_label">
         <property name="text">
          <string>&lt;b&gt;OR&lt;/b&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="fRunListFileName_label">
         <property name="text">
          <string>Run List File Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fRunListFileName_lineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>52</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fMsrFileExtension_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>msr File Extension</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>531</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="fExtension_label">
       <property name="text">
        <string>Extension</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="fMsrFileExtension_lineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fTemplateRunInput_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>551</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Template Run Input</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>531</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="fTemplateRunNumber_label">
       <property name="text">
        <string>Template Run Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fTemplateRunNumber_lineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="fChainFit_checkBox">
       <property name="text">
        <string>Chain Fit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fDataOutputFileName_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>551</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Ouput File Name</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>531</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="fDataOutputFileName_label">
       <property name="text">
        <string>Data Output File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fDataOutputFileName_lineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fOptions_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>551</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>540</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="fWriteDataHeader_checkBox">
         <property name="text">
          <string>Write Data Header</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fSummaryPresent_checkBox">
         <property name="text">
          <string>Summary Files Present</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fKeepMinuit2Output_checkBox">
         <property name="text">
          <string>Keep Minuit2 Output</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="fWriteColumnData_checkBox">
         <property name="text">
          <string>Write Column Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fRecreateDataFile_checkBox">
         <property name="text">
          <string>Recreate Data File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fOpenFilesAfterFitting_checkBox">
         <property name="text">
          <string>Open Files after Fitting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="fTitleFromData_checkBox">
         <property name="text">
          <string>Take Data File Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fCreateMsrFileOnly_checkBox">
         <property name="text">
          <string>Create msr-File only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fFitOnly_checkBox">
         <property name="text">
          <string>Fit Only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fGlobal_checkBox">
         <property name="text">
          <string>Global</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="fHelp_pushButton">
        <property name="text">
         <string>&amp;Help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>178</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fOk_pushButton">
        <property name="text">
         <string>&amp;OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fCancel_pushButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fFirst_lineEdit</tabstop>
  <tabstop>fLast_lineEdit</tabstop>
  <tabstop>fRunList_lineEdit</tabstop>
  <tabstop>fRunListFileName_lineEdit</tabstop>
  <tabstop>fMsrFileExtension_lineEdit</tabstop>
  <tabstop>fTemplateRunNumber_lineEdit</tabstop>
  <tabstop>fChainFit_checkBox</tabstop>
  <tabstop>fDataOutputFileName_lineEdit</tabstop>
  <tabstop>fWriteDataHeader_checkBox</tabstop>
  <tabstop>fSummaryPresent_checkBox</tabstop>
  <tabstop>fKeepMinuit2Output_checkBox</tabstop>
  <tabstop>fWriteColumnData_checkBox</tabstop>
  <tabstop>fRecreateDataFile_checkBox</tabstop>
  <tabstop>fOpenFilesAfterFitting_checkBox</tabstop>
  <tabstop>fTitleFromData_checkBox</tabstop>
  <tabstop>fCreateMsrFileOnly_checkBox</tabstop>
  <tabstop>fFitOnly_checkBox</tabstop>
  <tabstop>fOk_pushButton</tabstop>
  <tabstop>fCancel_pushButton</tabstop>
  <tabstop>fHelp_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../musredit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fCancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOk_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fHelp_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>helpContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fFirst_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>runFirstEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fLast_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>runLastEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fRunList_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>runListEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fRunListFileName_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>runListFileNameEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fTemplateRunNumber_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>templateRunEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fFitOnly_checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>fitOnlyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fCreateMsrFileOnly_checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PMsr2DataDialog</receiver>
   <slot>createMsrFileOnlyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>helpContent()</slot>
  <slot>fitOnlyChanged(int)</slot>
  <slot>createMsrFileOnlyChanged(int)</slot>
  <slot>runFirstEntered(QString)</slot>
  <slot>runLastEntered(QString)</slot>
  <slot>runListEntered(QString)</slot>
  <slot>runListFileNameEntered(QString)</slot>
  <slot>templateRunEntered(QString)</slot>
  <slot>globalChanged(int)</slot>
 </slots>
</ui>
